guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,400 @@
+   caption = "The user contact page allows the user to add labels to their receive static address before sharing it."
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Receiving flows are likely to be used less often since static addresses can be safely reused. If the receiver has publicly posted their static address, they will receive payments without any interaction with the sender. Therefore, applications should encourage users to add labels or a contact whenever they actively share a static address with a sender.
+
+Applications should also allow receivers to share an on-chain address itself in case the sender's wallet cannot send to static addresses.
+
+Some places where people can share static addresses (and add labels) include:
+- Social media pages
+- Fundraising websites
+- Exchanges
+- Email signatures
+
+</div>
+
+
+The tradeoff with silent payments for all their benefits is a higher blockchain scanning requirement. This scanning process is more computation-intensive and time-consuming than popular BIP-32 wallets/addresses. Mobile wallet users are likely to face a noticeable delay in the detecting payments once they come online. While the scanning is takes place, applications should show progress and estimated time to complete the scanning process.
+
+
+## Backup & Recovery
+
+### Backup
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/backup-wallet-birthday.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/backup-wallet-birthday@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/backup-wallet-birthday.png"
+   width = 250
+   height = 522
+   alt-text = "A backup flow screen with a calendar visual highlighting the wallet creation date and mentioning it prominently."
+   caption = "Display wallet birthday as a part of the backup flow and explain the benefit of storing it."
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Like [lightning wallets](/guide/daily-spending-wallet/backup-and-recovery/cloud-backup/), on-chain wallets with robust silent payments functionality require file backups (cloud or offline). This is because such wallets have valuable [user metadata](/guide/daily-spending-wallet/backup-and-recovery/landing-page/#user-metadata) such as:
+- scan and/or spend keys
+- [tweak data](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki#user-content-Tweak_Data)
+- [labels](/guide/how-it-works/silent-payments/#labels)
+- contacts
+
+Allowing the user to manually note some backup information in case the user misplaces the backup file or recovery application does not support backup files. Like regular wallets, silent payment wallets can also be backed up & restored with the recovery phrase. However, it may result in longer recovery times as well as loss of valuable metadata.
```

Comment:
```suggestion
Applications should allow users to manually note backup information in case they misplace the backup file or the recovery application does not support backup files at all. Like regular wallets, silent payment wallets can also be backed up with restored using a recovery phrase. However, this method may result in longer recovery times as well as loss of valuable metadata.
```